labview/labview_rabbitmq.c

```diff
@@ -196,19 +271,49 @@ int lv_amqp_login(int64_t conn_intptr, char *host, int port, int timeout_sec, ch
 
 
 LABVIEW_PUBLIC_FUNCTION
-int lv_amqp_basic_publish(int64_t conn_intptr, uint16_t  channel, char *exchange, char *routingkey, char *messagebody, LStrHandle error_description) {
+int lv_amqp_basic_publish(int64_t conn_intptr, uint16_t  channel, char *exchange, char *routingkey, char *cheaders_key, char *cheaders_value, char *messagebody, LStrHandle error_description) {
```

Comment:
inconsistent indentation in this function makes it difficult to read the code, can we apply a linter or so?

Reply:
I will look for it. I can see that the code in vs code and in GitHub review page looks different.

Reply:
it should look better now.

Reply:
i will look again after lunch ;)